fdm-app/app/components/custom/fertilizer-applications/list.tsx

```
@@ -0,0 +1,71 @@
+import { Button } from "@/components/ui/button"
+import { format } from "date-fns"
+import { LoadingSpinner } from "../loadingspinner"
+import type { FertilizerApplication } from "./types.d"
+
+export function FertilizerApplicationsList({
+    fertilizerApplications,
+    fetcher
+}: {
+    fertilizerApplications: FertilizerApplication[]
+    fetcher: any
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)